src/h5.cpp

```diff
@@ -181,19 +176,17 @@ append_ret h5d::append_data_1d(T const *data, hsize_t nlen) {
       sext.at(i) = sext2.at(i);
     }
     auto t3 = CLK::now();
-    LOG(Sev::Debug, "h5d::append_data_1d set_extent: {} + {}",
-        std::chrono::duration_cast<MS>(t2 - t1).count(),
-        std::chrono::duration_cast<MS>(t3 - t2).count());
+    Logger->trace("h5d::append_data_1d set_extent: {} + {}",
+                  std::chrono::duration_cast<MS>(t2 - t1).count(),
+                  std::chrono::duration_cast<MS>(t3 - t2).count());
   }
 
-  if (log_level >= 9) {
-    LOG(Sev::Debug, "try to get the dsp dims:");
-    auto sext = DSPTgt.current_dimensions();
-    auto smax = DSPTgt.maximum_dimensions();
-    for (int i1 = 0; i1 < ndims; ++i1) {
-      LOG(Sev::Debug, "dimensions: {:20} {}: {:21} {:21}", Name, i1,
-          sext.at(i1), smax.at(i1));
-    }
+  Logger->trace("try to get the dsp dims:");
```

Comment:
Can you keep the log level check? This function is called a lot so we should be careful about any potential performance hits.